### PR TITLE
fix: show Fees Vary when Fiatconnect quotes return undefined for fee

### PR DESCRIPTION
**Pull request by XrayBravoGolf**
### Description

A bug was discovered that when FiatConnect returns a quote without fees (allowed since fees are optional in the quote response), the select provider screen shows that provider quote with fee of NaN.
This PR fixes it by replacing such fee with "Fees Vary".
This shall work with both single and multiple quotes; when there are multiple quotes, "Best Rate" is shown only when there is one with known fee.

### Other changes

None

### Tested

Tests are re-written to check the UI for "FeesVary" when for fee of `undefined` is present in a single quote or among multiple quotes.

### Related issues

- Fixes #[ACT-450](https://linear.app/valora/issue/ACT-450/bugfix-nan-when-fee-is-undefined)

### Backwards compatibility

Changes are backwards compatible because only visual interface changes are involved, and no schema or state handling were modified.

### CODE REVIEW

**Comment by XrayBravoGolf** [comment]
Screenshots for before and after:
Simulator Screen Shot - iPhone 13 - 2022-09-13 at 03 01 14
Simulator Screen Shot - iPhone 13 - 2022-09-13 at 03 01 20
88977c3c18f1.png)
Simulator Screen Shot - iPhone 13 - 2022-09-13 at 11 45 27

**Comment by XrayBravoGolf** [comment]
Screenshot for single quote:
Simulator Screen Shot - iPhone 13 - 2022-09-13 at 14 30 11

**Comment by XrayBravoGolf** [comment]
Screenshot for all quotes having no fee returned
Simulator Screen Shot - iPhone 13 - 2022-09-13 at 14 36 49

File: src/fiatExchanges/quotes/normalizeQuotes.test.ts [hunk 1]
```diff
@@ -38,6 +42,29 @@ describe('normalizeQuotes', () => {
       ['Moonpay', 10],
     ])
   })
+
+  it('sorts FiatConnect quotes with no fee returned at the end of quotes', () => {
```
Comment (cajubelt) [inline src/fiatExchanges/quotes/normalizeQuotes.test.ts:46]: this is fine, but there should also be a test for the higher-level behavior you were trying for with the bugfix

**Review by cajubelt — COMMENTED** [review]
looks good, but needs tests

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Android Alfajores Release Build V1.41.0 & Test Flight Alfajores Build V1.41.0 and updated status in test rails.

**Observations**:
- User is not able to see Nan on fees section on Select Withdrawal method using cUSD, cEUR, CELO for fiat connect flow.

**Test rail link:**
**Android 10:** https://valoraapp.testrail.io/index.php?/tests/view/74600
**Android 12:** https://valoraapp.testrail.io/index.php?/tests/view/74631
**iOS 12:** https://valoraapp.testrail.io/index.php?/tests/view/74662

**Devices**: Google Pixel 4a (12.0), Galaxy M31(10.0), iPhone 6+(12.5.5)

**CC: @XrayBravoGolf 

Thanks**..!!

**Pull request merged**